Networking/Networking/Extensions/Data+Woo.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+extension Data {
+    /// Checks whether the JSON data has a `data` key at the root.
+    ///
+    var hasDataEnvelope: Bool {
```

Comment:
❓  it feels a bit odd for any kind of `Data` to have this "data envelope" extension, like for non-Jetpack/WPCOM requests or data that aren't related to API response. how about having this helper in `Mapper`?

Reply:
Good point! I Moved the extension to `Mapper` in 5a20e54cc47428b51b1bf426aec4ea72678b941d.